voter/management/commands/voter_fetch_snapshot.py

```diff
@@ -131,8 +131,10 @@ def handle(self, *args, **options):
             objects = s3client.list_objects(Bucket='dl.ncsbe.gov',Prefix='data/Snapshots/')
             filenames=[]
             for i in objects['Contents']:
-                if i['Key'].split('/')[-1].endswith('.zip'):
-                    filenames.append(i['Key'].split('/')[-1])
+                fn = i['Key'].split('/')[-1]
```

Comment:
I like this refactor (though I'd prefer to spell out `filename`)

Reply:
I'm actually going to counter this with a concern that the variables `filename` and `filenames` are then far too close.

Even as I went to make this change I almost typoed `filenames.append(filenames)`

Reply:
okay :( fn looks like 'function' to me